### ISSUE TITLE
FallbackLocale not working

### ISSUE THREAD

**Issue by Quadrubo** [xiCO2k/laravel-vue-i18n#87]
The FallbackLocale option is not working correctly.

```php
// config/app.php
'locale' => 'de',

 'fallback_locale' => 'en',
 ```
 
```php
// resources/lang/en/app.php
return [

    'jetstream' => [

        'Forgot Password' => 'Forgot Password',
        'Two-factor Confirmation' => 'Two-factor Confirmation',
        'Email Verification' => 'Email Verification',

    ],

];
```

```php
// resources/lang/de/app.php
<?php

return [

    'jetstream' => [

        // 'Forgot Password' => 'Passwort vergessen',
        'Two-factor Confirmation' => 'Zwei-Faktor-Authentifizierung',
        'Email Verification' => 'E-Mail-Verifizierung',

    ],

];
```
```js
// resources/js/app.js
.use(i18nVue, {
    lang: 'de',
    fallbackLang: 'en',
    resolve: async lang => {
        const langs = import.meta.glob('../lang/*.json');
        return await langs[`../lang/${lang}.json`]();
    }
})
```
I would expect the `trans` function to use the fallbackLocale. Instead it echoes out the translation string.
```js
// 'Forgot Password' is set in de/app.php, locale = 'de', fallbackLocale = 'en'
console.log(trans('app.jetstream.Forgot Password')); // -> Passwort vergessen

// 'Forgot Password' is not set in de/app.php, locale = 'de', fallbackLocale = 'en'
console.log(trans('app.jetstream.Forgot Password')); // -> app.jetstream.Forgot Password
```

**Comment by xiCO2k** [comment]
Will review that one thanks for the issue.

**Comment by lenssoft** [comment]
I am experiencing the same issue. I think it could be fixed in setLanguege method.
<img width="877" alt="image" src="https://user-images.githubusercontent.com/9431885/221153453-ccde2444-83a7-4820-8302-e36e884c2b98.png">

Instead of setting `this.activeMessages[key] = null` better use the fallbackLang and those messages to be kept in `fallbackMessages`
`this.activeMessages[key] = this.fallbackMessages[key] || null`

It would comet with the drawback that fallback lang must always be loaded and extra memory usage due to keeping them in `fallbackMessages`

**Comment by xiCO2k** [comment]
Hey @lenssoft if you can, please submit a PR for that, and I will review it.

Thanks.

**Comment by xiCO2k** [comment]
Fixed on #103
